src/scss/components/optional/_cookie-warning.scss

```diff
@@ -1,65 +1,51 @@
 @mixin cookieWarning() {
-	.c-cookieWarning {
-	    background-color: $grey--darkest;
-	    position: fixed;
-	    bottom: 0;
-		width: 100%;
-		z-index: zIndex(high);
-	}
+    .c-cookieWarning {
+        background-color: $grey--darkest;
+        position: fixed;
+        bottom: 0;
+        width: 100%;
+        z-index: zIndex(high);
 
-	.c-cookieWarning-inner {
-	    width: 940px;
-	    margin: 0 auto;
-	    padding: spacing() 0;
-	    overflow: hidden;
+        & p {
```

Comment:
technically you don't need an ampersand but it works 👍

Reply:
True! Personal preference... although it would be good to have a consistent style on this.